.github/workflows/plan-examples.py

```diff
@@ -1,30 +0,0 @@
-import json
```

Comment:
this lives here https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/.github/scripts/plan-examples.py